GOVERNANCE.md

```diff
@@ -105,3 +108,60 @@ request a vote be taken.
 Most votes require a simple majority of all Maintainers to succeed. Maintainers
 can be removed by a 2/3 majority vote of all Maintainers, and changes to this
 Governance require a 2/3 vote of all Maintainers.
+
+## SIGs
+
+The KubeVirt project is organized primarily into Special Interest Groups, or SIGs.
+Each SIG is comprised of members from multiple companies and organizations, with a
+common purpose of advancing the project with respect to a specific topic, such as
+Networking or Storage. Our goal is to enable a distributed decision structure
+and code ownership, as well as providing focused forums for getting work done,
+making decisions, and onboarding new contributors. Every identifiable subpart of
+the project (e.g., github org, repository, subdirectory, API, test, issue, PR)
+is intended to be owned by some SIG.
+
+Our SIGs define mission and scope via their charters, ownership of code via
+[OWNERS](https://www.kubernetes.dev/docs/guide/owners/), and members and roles in
+[sigs.yaml].
+
+For more details see also the [Kubernetes SIGs] model.
+
+Examples:
+* [sig-network - charter](./sig-network/charter.md)
+* [sig-network - kubevirt/kubevirt OWNERS_ALIASES](https://github.com/kubevirt/kubevirt/blob/a7e0311d8704663351abd4bc9bbc8511753d2838/OWNERS_ALIASES#L60)
+* [sig-ci - sigs.yaml](https://github.com/kubevirt/community/blob/4f63a79c0ed810aa332cd6716d4986001d28bcd7/sigs.yaml#L119)
+
+### Subprojects
+
+Specific work efforts within SIGs are divided into subprojects. Every part of the
+KubeVirt code and documentation must be owned by some subproject. Some [SIGs](#sigs)
+may have a single subproject, but many SIGs have multiple significant subprojects
+with distinct (though sometimes overlapping) sets of contributors and owners, who
+act as subproject’s technical leaders: responsible for vision and direction and
+overall design, choose/approve design proposals approvers, field technical
+escalations, etc.
+
+For more details see also the [Kubernetes Subprojects] model.
+
+## Working Groups
+
+We need community rallying points to facilitate discussions/work regarding topics
+that are short-lived or that span multiple SIGs.
+
+Working groups are primarily used to facilitate topics of discussion that are in
+scope for KubeVirt but that cross SIG lines.
+
+ Our WGs define mission and scope via their charter, and members and roles in
+ [sigs.yaml].
+
+For more details see also the [Kubernetes WGs] model.
+
+## Roles and Organization Management
+
+All our SIGs and WGs follow the Roles and Organization Management outlined in [OARP]
+
+[Kubernetes SIGs]: https://github.com/kubernetes/community/blob/master/governance.md#sigs
+[Kubernetes Subprojects]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
+[Kubernetes WGs]: https://github.com/kubernetes/community/blob/master/governance.md#working-groups
+[OARP]: https://stumblingabout.com/tag/oarp/
```

Comment:
I like the slide this links to I'm not sure it's a good source to link to: a short-lived blog that hasn't been updated in 11 years. Unfortunately, I also can't find an alternative for OARP, and I wonder if it's because it's a somewhat bespoke modification of RACI etc since the blog owner was a strategy consultant. 
The linked blogpost doesn't really go into much detail either beyond the slide. 
What about linking to our Community membership doc? https://github.com/kubevirt/community/blob/main/membership_policy.md
In the future we could add to that doc with a diagram showing the relationship between the different roles.  WDYT?

Reply:
Yeah, good point. I agree that the article might vanish or even be modified at some later point in the future, so I am going to link to the membership policy for now, knowing that the roles are still missing - I'll create a followup PR that updates the membership policy with the additional roles.

@aburdenthehand would that work for you?

Reply:
@dhiller Yeah, I think that will work better. Thanks Daniel!